terraform/modules/aws-brig-prekey-lock-event-queue-email-sending/resources.iam.tf

```diff
@@ -0,0 +1,57 @@
+resource "aws_iam_user" "brig" {
+  name          = "${var.environment}-brig-services"
+  force_destroy = true
```

Comment:
We called the user `<env>-<service>-full-access` [here](https://github.com/wireapp/wire-server-deploy/blob/develop/terraform/modules/aws-gundeck-push-notifications/resources.iam.tf#L2). Can we keep it like that?